var/spack/repos/builtin/packages/bear/package.py

```diff
@@ -15,5 +15,9 @@ class Bear(CMakePackage):
     version('2.2.0', sha256='6bd61a6d64a24a61eab17e7f2950e688820c72635e1cf7ea8ea7bf9482f3b612')
     version('2.0.4', sha256='33ea117b09068aa2cd59c0f0f7535ad82c5ee473133779f1cc20f6f99793a63e')
```

Comment:
```suggestion
    version('2.0.4', sha256='33ea117b09068aa2cd59c0f0f7535ad82c5ee473133779f1cc20f6f99793a63e')
    version('3.0.0', sha256='7b68aad69e887d945ad20f8e9f3a8c33cf2d59cc80da7e52d931d8c685fe2f79')
```

Reply:
The convention is to have newer versions appear *before* older ones, so the directive should be added *before* `2.0.4`.